cmake/cli_input.cmake

```diff
@@ -28,6 +28,7 @@ set (unit-test false)
 set (MANUFACTURER_TOOLKIT false)
 set (STORAGE true)
 set (BOARD NUCLEO_F767ZI)
+set(BLOB_PATH .)
```

Comment:
Add a space after set for uniformity.
set( => set (

Reply:
@trbehera fixed this. please re-review.